app/expirynotify.go

```diff
@@ -17,12 +17,8 @@ const (
 
 // NotifySessionsExpired is called periodically from the job server to notify any mobile sessions that have expired.
 func (a *App) NotifySessionsExpired() error {
-	if *a.Config().EmailSettings.SendPushNotifications {
-		pushServer := *a.Config().EmailSettings.PushNotificationServer
-		if license := a.ch.srv.License(); pushServer == model.MHPNS && (license == nil || !*license.Features.MHPNS) {
-			mlog.Warn("Push notifications are disabled. Go to System Console > Notifications > Mobile Push to enable them.")
-			return nil
-		}
+	if !a.canSendPushNotifications() {
```

Comment:
Another behavior change here. Before, we only checked the license if the sendPushNotifications setting was set. But if it was not set, we were sending the session expired notifications.

Now we don't send these notifications if the sendPushNotifications is set to false.

Reply:
CC @enahum

Reply:
Great find, I think this was a clear bug.